client/app/scripts/superdesk-archive/views/metadata-view.html

```diff
@@ -78,7 +78,7 @@
     <!-- category details -->
     <dl ng-if="item['anpa-category']">
         <dt translate>ANPA category</dt>
-        <dd>{{ item['anpa-category'].name }}</dd>
```

Comment:
since we are changing type of the field from dict to list of dict. Can we change the field name to anpa_category instead of anpa-category?